lib/lib.dom.iterable.d.ts

```diff
@@ -107,6 +90,31 @@ interface NodeListOf<TNode extends Node> {
     [Symbol.iterator](): IterableIterator<TNode>;
 }
 
+interface HTMLCollectionBase {
+    [Symbol.iterator](): IterableIterator<Element>;
+}
+
+interface HTMLCollectionOf<T extends Element> {
+    [Symbol.iterator](): IterableIterator<T>;
+}
+
+interface FormData {
+    /**
+     * Returns an array of key, value pairs for every entry in the list
+     */
+    entries(): IterableIterator<[string, string | File]>;
+    /**
+     * Returns a list of keys in the list
+     */
+    keys(): IterableIterator<string>;
+    /**
+     * Returns a list of values in the list
+     */
+    values(): IterableIterator<string | File>;
+
+    [Symbol.iterator](): IterableIterator<string | File>;
+}
```

Comment:
(out of scope for this PR but) this pattern repeats often enough that maybe there should be a virtual interface in `lib.es2015.iterable.d.ts` for this 🤔 

```ts
interface IterableCollection<K, V = K, I = V> {
  entries(): IterableIterator<[K, V]>;
  keys(): IterableIterator<K>;
  values(): IterableIterator<V>;
  [Symbol.Iterator](): IterableIterator<I>
}
```